### PR TITLE
adding support for WindowsEvent in Sysmon parsers

### PR DESCRIPTION
**Pull request by YuvalNaor**
Fixes #

## Proposed Changes

  - supporting Sysmon in WindowsEvent table across relevant schemas!
  -
  -

### CODE REVIEW

File: Parsers/ASimFileEvent/ARM/FileEventMicrosoftSysmonFileCreated/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Windows Sysmon ASIM FileEvent Normalization Parser
 
-This template deploys the ASIM FileEvent schema parser for Windows Sysmon. The parser is a part of the Azure Sentinel Information Model.
+This template deploys the ASIM FileEvent schema parser for Windows Sysmon. The parser is a part of the Azure Sentinel Information Mode.
```
Comment (YaronFruchtmann) [inline Parsers/ASimFileEvent/ARM/FileEventMicrosoftSysmonFileCreated/README.md:3]: Model

File: Parsers/ASimFileEvent/ARM/FileEventMicrosoftSysmonFileDeleted/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Windows Sysmon ASIM FileEvent Normalization Parser
 
-This template deploys the ASIM FileEvent schema parser for Windows Sysmon. The parser is a part of the Azure Sentinel Information Model.
+This template deploys the ASIM FileEvent schema parser for Windows Sysmon. The parser is a part of the Azure Sentinel Information Mode.
```
Comment (YaronFruchtmann) [inline Parsers/ASimFileEvent/ARM/FileEventMicrosoftSysmonFileDeleted/README.md:4]: model

File: Parsers/ASimProcessEvent/ARM/ProcessEventMicrosoftSysmonCreate/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Microsoft Windows Events Sysmon ASIM ProcessEvent Normalization Parser
 
-This template deploys the ASIM ProcessEvent schema parser for Microsoft Windows Events Sysmon. The parser is a part of the Azure Sentinel Information Model.
+This template deploys the ASIM ProcessEvent schema parser for Microsoft Windows Events Sysmon. The parser is a part of the Azure Sentinel Information Mode.
```
Comment (YaronFruchtmann) [inline Parsers/ASimProcessEvent/ARM/ProcessEventMicrosoftSysmonCreate/README.md:3]: Model.... :)

File: Parsers/ASimRegistry/ARM/RegistryEventMicrosoftSysmon/README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Microsoft Windows Events Sysmon ASIM RegistryEvent Normalization Parser
 
-This template deploys the Microsoft Windows Events Sysmon ASIM RegistryEvent schema parser. The parser is a part of the Azure Sentinel Information Model.
+This template deploys the ASIM RegistryEvent schema parser for Microsoft Windows Events Sysmon. The parser is a part of the Azure Sentinel Information Mode.
```
Comment (YaronFruchtmann) [inline Parsers/ASimRegistry/ARM/RegistryEventMicrosoftSysmon/README.md:3]: l

**Comment by YuvalNaor** [comment]
@YaronFruchtmann @oshezaf

File: Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileDeleted.yaml [hunk 5]
```diff
@@ -12,55 +12,92 @@ References:
   Link: https://aka.ms/AzSentinelFileEventDoc 
 - Title: ASIM
   Link: https://aka.ms/AzSentinelNormalization
-Description: ASIM Sysmon/Event File Deletion Event Parser (event number 23)
+Description: ASIM Sysmon/Event File Deletion Event Parser (event number 23) from "Event" and "WindowsEvent" tables.
```
Comment (oshezaf) [inline Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileDeleted.yaml:15]: Should remove "/Event" as you did above.

**Review by oshezaf — CHANGES_REQUESTED** [review]
One small change.

File: Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileDeleted.yaml (hunk 5 above)
Comment (YuvalNaor) [reply Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileDeleted.yaml]: Done

**Review by oshezaf — APPROVED** [review]

**Pull request merged**